core/src/main/java/hera/core/HeraUtil.java

```diff
@@ -47,23 +45,30 @@ public static Command getCommandFromMessage(String message, String prefix, Guild
 		// add modularisation stuff here (get module for guild + command from STORE and check permission for it)
 
 		boolean isOwner = !STORE.owners().getAll().stream()
-			.filter(owner -> owner.getUser().equals(member.getId().asLong()))
-			.collect(Collectors.toList()).isEmpty();
+				.filter(owner -> owner.getUser().equals(member.getId().asLong()))
+				.collect(Collectors.toList()).isEmpty();
 
 		if (isOwner) return Mono.just(true);
 
-		if (command.isAdmin()) {
-			return member.getBasePermissions()
-					.filter(permissions -> permissions.contains(Permission.ADMINISTRATOR))
-					.hasElement()
-					.flatMap(exist -> {
-						if (exist) return Mono.just(true);
-						return channel.createMessage(LOCALISATION_PERMISSION_ERROR.getValue()).flatMap(c ->  Mono.just(false));
-					});
-		}
-		else {
-			return Mono.just(true);
+		List<ModuleSettings> msList = STORE.moduleSettings().forModule(guild.getId().asLong(), command.getId());
+		ModuleSettings ms = msList.size() > 0 ? msList.get(0) : null;
```

Comment:
You can ignore this if you don't like it, but I prefer `msList.isEmpty()` over checking the lists size.